src/Fist/Http/src/Request.php

```diff
@@ -4,55 +4,55 @@
 
 class Request
 {
-	protected $server;
+    protected $server;
 
-	protected $baseUrl;
+    protected $baseUrl;
 
-	protected $path;
+    protected $path;
 
-	protected $requestUri;
+    protected $requestUri;
 
     public function __construct(array $query = [], array $request = [], array $cookies = [], array $files = [], array $server = [], $content = null)
     {
         $this->initialize($query, $request, $cookies, $files, $server, $content);
     }
 
-	public static function createFromGlobals()
-	{
-		return new static($_GET, $_POST, $_COOKIE, $_FILES, $_SERVER);
-	}
+    public static function createFromGlobals()
```

Comment:
__[phpmd]__
* createFromGlobals accesses the super-global variable $_GET.
* createFromGlobals accesses the super-global variable $_POST.
* createFromGlobals accesses the super-global variable $_COOKIE.
* createFromGlobals accesses the super-global variable $_FILES.
* createFromGlobals accesses the super-global variable $_SERVER.

<!-- Comment by [SideCI](https://sideci.com) -->